src/Component/Console/src/Helper/Table.js

```diff
@@ -0,0 +1,633 @@
+        let content = __jymfony.sprintf(style.getCellRowContentFormat(), cell);
+
+        return __jymfony.sprintf(cellFormat, str_pad(content, width, style.getPaddingChar(), style.getPadType()));
+    }
+
+    /**
+     * Calculate number of columns for this table.
+     *
+     * @private
+     */
+    _calculateNumberOfColumns() {
+        if (this._numberOfColumns) {
+            return;
+        }
+
+        let columns = [0];
+
+        for (let row of [].concat(this._headers, this._rows)) {
+            if (row instanceof TableSeparator) {
+                continue;
+            }
+
+            columns.push(this._getNumberOfColumns(row));
+        }
+
+        this._numberOfColumns = Math.max(...columns);
+    }
+
+    /**
+     * @param {Array} rows
+     *
+     * @returns {Array}
+     *
+     * @private
+     */
+    _buildTableRows(rows) {
+        let unmergedRows = [];
+
+        for (let rowNumber = 0; rowNumber < rows.length; ++rowNumber) {
+            let rows = this._fillNextRows(rows, rowNumber);
+
+            // Remove any new line breaks and replace it with a new line
+            for (let [ column, cell ] of __jymfony.getEntries(rows[rowNumber])) {
+                if (-1 === cell.indexOf("\n")) {
+                    continue;
+                }
+
+                let lines = cell.replace("\n", "<fg=default;bg=default>\n</>").split("\n");
+                for (let [ lineKey, line ] of __jymfony.getEntries(lines)) {
+                    if (cell instanceof TableCell) {
+                        line = new TableCell(line, { 'colspan': cell.getColspan() });
+                    }
+
+                    if (0 === lineKey) {
+                        rows[rowNumber][column] = line;
+                    } else {
+                        unmergedRows[rowNumber][lineKey][column] = line;
+                    }
+                }
+            }
+        }
+
+        let tableRows = [];
+        for (let rowNumber = 0; rowNumber < rows.length; ++rowNumber) {
+            tableRows.push(this._fillCells(rows[rowNumber]));
+
+            if (!!unmergedRows[rowNumber]) {
+                tableRows = [].concat(tableRows, unmergedRows[rowNumber]);
+            }
+        }
+
+        return tableRows;
+    }
+
+    /**
+     * fill rows that contains rowspan > 1.
+     *
+     * @param {Array} rows
+     * @param {int} line
+     *
+     * @return {Array}
+     *
+     * @private
+     */
+    _fillNextRows(rows, line) {
+        let unmergedRows = [];
+
+        for (const [ column, cell ] of __jymfony.getEntries(rows[line])) {
+            if (cell instanceof TableCell && cell.getRowspan() > 1) {
+                let nbLines = cell.getRowspan() - 1;
+                let lines = [cell];
+
+                if (-1 < cell.indexOf("\n")) {
+                    let lines = cell.replace("\n", "<fg=default;bg=default>\n</>").split("\n");
+                    nbLines = lines.length > nbLines ? __jymfony.substr_count(cell, "\n") : nbLines;
```

Comment:
No need to import `substr_count`. Use `cell.split("\n").length - 1` instead.

Reply:
:+1: